### PR TITLE
Add an OpenVINO export target and inference runtime

### LINKED ISSUE #798: Add an OpenVINO export target and inference runtime for token-classification families
## Summary
Intel CPUs/iGPUs/NPUs are the dominant edge fleet in clinics, and OpenVINO is the fastest path there. OpenMed has no OpenVINO export or runtime, so those devices fall back to generic CPU. An IR export plus a session wrapper unlocks high-throughput on-device inference on Intel hardware.

## Scope
- [ ] An OpenVINO IR export profile in openmed/onnx/openvino_export.py converting token-classification families from ONNX
- [ ] An inference session wrapper selecting CPU/GPU/NPU device and returning logits
- [ ] INT8 post-training quantization via NNCF with a per-family recall-delta certification (G4 gate)
- [ ] A per-device throughput/latency record in the benchmark format

## Acceptance criteria
- [ ] Exported IR runs a synthetic note and matches the ONNX reference spans within a fixed tolerance
- [ ] INT8 export passes the recall-delta gate on the synthetic eval or is rejected fail-closed
- [ ] Device selection falls back deterministically when a requested device is absent

## Out of scope
- Bundling Intel proprietary drivers
- Training
- Multimodal paths

## Files
- openmed/onnx/openvino_export.py
- openmed/onnx/openvino_session.py
- tests/unit/onnx/test_openvino_export.py
- docs/runtimes/openvino.md

---
Task: OM-441  ·  Milestone: v1.8  ·  Priority: P1  ·  Size: L
Depends on: OM-018, OM-371  ·  Blocks: —
Roadmap: v1.8/v1.9 expansion (post-1.7)
Spec: PLANS/V2/EXECUTION/tasks/OM-441.md

### PR DESCRIPTION
**Pull request by maziyarpanahi**
Closes #798.

Adds an OpenVINO IR export profile for ONNX token-classification artifacts, a deterministic device-selecting session wrapper, fail-closed INT8 NNCF quantization with G4 recall-delta evidence, and BenchmarkReport-compatible per-device latency and throughput records.

Validation:
- .venv/bin/python -m pytest tests/ -q
- make lint
- make format-check
- uv run mkdocs build --strict
- uv lock --check

### CODE REVIEW

**Comment by maziyarpanahi** [comment]
Maintainer validation complete for the refreshed branch.

- Resolved the OpenVINO/ONNX refresh against current master, including the lockfile and license-policy updates.
- Ran uv lock --check.
- Ran python3 scripts/release/check_license_policy.py.
- Ran focused Ruff over the ONNX/OpenVINO conversion/runtime paths and tests.
- Ran focused pytest for the OpenVINO export, ONNX conversion, and Android INT8 ONNX tests: 22 passed.
- Ran uv run --extra docs mkdocs build --strict.
- Ran git diff --check.
- GitHub checks are green.

Merging with the reviewed head commit.
